FrameworkSystem/Client/SystemAdministratorClientCLI.py

```diff
@@ -473,21 +476,64 @@ def do_uninstall( self, args ):
 
         usage:
 
-          uninstall <system> <component>
+          uninstall db <database>
+          uninstall <-f ForceLogUninstall> <system> <component>
     """
     argss = args.split()
-    if not argss or len(argss) != 2:
+    if not argss:
       print self.do_uninstall.__doc__
       return
     
-    system,component = argss
-    client = SystemAdministratorClient( self.host, self.port )
-    result = client.uninstallComponent( system, component )
-    if not result['OK']:
-      print "Error:", result['Message']
+    option = argss[0]
+    if option == 'db':
+      component = argss[1]
+      client = SystemAdministratorClient( self.host, self.port )
+
+      result = client.uninstallDatabase( component )
+      if not result[ 'OK' ]:
+        self.__errMsg( result[ 'Message' ] )
+      else:
+        print "Successfully uninstalled %s" % ( component )
```

Comment:
In general we should not use print, not even in scripts. I understand several scripts (including this one) use it, but it would be better using gLogger like everywhere else.